src/services/JiraService.ts

```diff
@@ -140,7 +141,10 @@ export async function postJiraComment(taskId: string, comment: string) {
     data: {
       body: comment
     },
-  }).then((response) => response.data);
+  })
+    .then(createSuccessHandler({ title: 'Success!', body: 'Internal comment saved' }))
+    .then((response) => response.data)
+    .catch(createCatchHandler('Error saving internal Jira comment'));
```

Comment:
Why are we handling errors here? This is the only service that handles error internally, making the use a bit confusing.

`createSuccessHandler` and `createCatchHandler` uses the `toast` store to show the message to the screen, I think services should be side-effects free, so they can be used anywhere, is up to the caller to handle the errors, so they can decide what to do (fail silently, retry on error...).